### PR TITLE
Syndicate Airlocks cannot be scanned

### LINKED ISSUE #6913: syndicate doors can be scanned
### Labels

[BUG] [MINOR]

### Description

Doors at the listening post can be scanned (yes this is a bug as they are indestructible).

### Steps to reproduce

1. go to the listening outpost
2. scan the door

### Expected Behavior

not being able to scan the door or the door not retaining the immunity to bombs and meteors

### Additional Information & Screenshots

_No response_

### PR DESCRIPTION
**Pull request by Zonespace27**
[Bug]
## About the PR 
Title.

## Why's this needed? 
Being able to scan and reproduce near-invincible doors by default is bad.
fixes #6913

### CODE REVIEW

File: code/obj/machinery/door/airlock.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	isWireCut - returns 1 if that wire (e.g. AIRLOCK_WIRE_DOOR_BOLTS) is cut, or 0 if not
 	canAIControl - 1 if the AI can control the airlock, 0 if not (then check canAIHack to see if it can hack in)
 	canAIHack - 1 if the AI can hack into the airlock to recover control, 0 if not. Also returns 0 if the AI does not *need* to hack it.
-	arePowerSystemsOn - 1 if the main or backup power are functioning, 0 if not. Does not check whether the power grid is charged or an APC has equipment on or anything like that. (Check (status & NOPOWER) for that)
+	arePowerSystemsOn - 1 if the main or backup power are functioning, 0 if not. Does not check whether the power grid is charged or anAPC has equipment on or anything like that. (Check (status & NOPOWER) for that)m
```
Comment (Zonespace27) [inline code/obj/machinery/door/airlock.dm:13]:
```suggestion
	arePowerSystemsOn - 1 if the main or backup power are functioning, 0 if not. Does not check whether the power grid is charged or an APC has equipment on or anything like that. (Check (status & NOPOWER) for that)m
```
fuuuuuuck

**Comment by flappybatpal** [comment]
I don't think these should be scannable at all. As you say, these are supposed to be infinite for a very specific reason.

This is my understanding and opinion, not a developers.

**Comment by penny64** [comment]
I was the last one in there and I'd suggest we just make constructed doors not invincible?

**Comment by ZWRh3** [comment]
> 

they arent even special in any way they are just hard to break and impossible to hack

**Comment by Sovexe** [comment]
Please also include centcom doors in this

**Comment by Zonespace27** [comment]
Done

**Review by Sovexe — APPROVED** [review]

**Pull request merged**